dashboards/component/chart/serializers.py

```diff
@@ -43,14 +40,12 @@ def get_data(self, *args, **kwargs) -> pd.DataFrame:
         return df
 
     def get_queryset(self, *args, **kwargs):
-        if self._meta.model is not None:
-            queryset = self._meta.model._default_manager.all()
-        else:
-            raise ImproperlyConfigured(
-                "%(self)s is missing a QuerySet. Define "
-                "%(self)s.model or override "
-                "%(self)s.get_queryset()." % {"self": self.__class__.__name__}
-            )
+        queryset = super().get_queryset(*args, **kwargs)
```

Comment:
you have removed all the code which sets the queryset up so this will break it. e.g. `queryset = self._meta.model._default_manager.all()`